Starr - Simple text corrections and button placement

dstarr opened the pull request.

code4clouds commented:
@dstarr  Would you mind adding a picture of the changes?

code4clouds commented on .gitignore:
```diff
@@ -362,4 +362,5 @@ src/.idea/.idea.Microsoft.Marketplace.SaaS.SDK/.idea/projectSettingsUpdater.xml
-appsettings.Development.json
\ No newline at end of file
+appsettings.Development.json
+/src/SaaS.SDK.PublisherSolution/appsettings.Development.json
```
@dstarr  Did you mean to add the development files here?  Should we have instructions on how to apply the debug settings on it?

dstarr replied:
> @dstarr Did you mean to add the development files here? Should we have instructions on how to apply the debug settings on it?

Sorry, but I am not exactly sure what you mean. I removed this file from Git as it shouldn't be here. It should be ignored. As for why, the version that I found on the main branch had client secrets and app IDs in it because someone checked it in.

dstarr replied:
Added dev appsettings file back with it also in .gitignore

dstarr commented:
> @dstarr Would you mind adding a picture of the changes?

Added Go to Subscriptions list.
![image](https://user-images.githubusercontent.com/64742/167189840-eda10ef6-e92b-4d4a-afcc-267f5a9f923f.png)

Added message at the top of page.
![image](https://user-images.githubusercontent.com/64742/167190032-72eb1d56-775d-4bee-b7e8-5879e0943761.png)

code4clouds commented:
@dstarr There are some merging conflicts, you may need to take care of. :-/

dstarr commented:
> @dstarr There are some merging conflicts, you may need to take care of. :-/

Merge issues resolved.

code4clouds commented on src/SaaS.SDK.PublisherSolution/Views/Home/ViewSubscriptionDetail.cshtml:
```diff
@@ -44,18 +44,6 @@
                             @Html.HiddenFor(s => s.SubscribeId)
                         </dd>
                     </dl>
-                    @*<div class="row pr-3 pt-3 mt-5 pb-3 text-right" style="background-color:#F2F2F2">
```
@dstarr  Why is this block removed?

dstarr commented:
> If there are other errors we want the script to stop, right?
Correct.

code4clouds commented on src/SaaS.SDK.PublisherSolution/appsettings.Development.json:
```diff
@@ -7,21 +7,21 @@
+    "granttype": "client_credentials",
+    "tenantid": "de9fb1cb-df80-4ff5-9977-4dfa053a74a7",
+    "clientid": "302464fb-a6ae-4ace-afc1-cc68ae6080ec",
+    "clientsecret": "M2JiYjU0NTgtMmQwOC00ODExLTllMzEtMDAwNmY4NTk0MDAw=",
```
@dstarr - You left your test env secrets.

code4clouds commented on src/SaaS.SDK.CustomerProvisioning/appsettings.Development.json:
```diff
@@ -5,5 +5,23 @@
+    "saasappurl": ""
+  },
+  "connectionstrings": {
+    "defaultconnection": "Data Source=tcp:ecodesaasacc00.database.windows.net,1433;Initial Catalog=AMPSaaSDB;User Id=adminlogin@ecodesaasacc00.database.windows.net;Password=password21!!;"
```
@dstarr  - creds in here as well.

code4clouds commented on src/SaaS.SDK.PublisherSolution/appsettings.Development.json:
```diff
@@ -7,21 +7,21 @@
+    "saasappurl": ""
+  },
+  "connectionstrings": {
+    "defaultconnection": "Data Source=tcp:ecodesaasacc00.database.windows.net,1433;Initial Catalog=AMPSaaSDB;User Id=adminlogin@ecodesaasacc00.database.windows.net;Password=password21!!;"
```
@dstarr  - creds here as well

code4clouds commented on src/SaaS.SDK.CustomerProvisioning/appsettings.Development.json:
```diff
@@ -5,5 +5,23 @@
+    "granttype": "client_credentials",
+    "tenantid": "de9fb1cb-df80-4ff5-9977-4dfa053a74a7",
+    "clientid": "302464fb-a6ae-4ace-afc1-cc68ae6080ec",
+    "clientsecret": "M2JiYjU0NTgtMmQwOC00ODExLTllMzEtMDAwNmY4NTk0MDAw=",
```
@dstarr Secret here too...

dstarr commented:
Moving these changes to smaller and more localized PRs.
